StortSpelProjekt/Engine/src/ECS/Components/Component.h

```diff
@@ -15,7 +15,11 @@ class Component
 	//Use render update for all things rendering. It will be using variable timestep
 	virtual void RenderUpdate(double dt);
 	//SceneInit
-	virtual void InitScene();
+	virtual void OnInitScene();
+	// Called on LoadScene for each component
+	virtual void OnLoadScene();
+	// Called on UnloadScene for each component
+	virtual void OnUnloadScene();
```

Comment:
Are you sure these aren't meant to be pure virtual? It might be good to remember people to implement these functions in their components.

Reply:
fixed